test/performance/odp_l2fwd.c

```diff
@@ -2299,14 +2281,13 @@ int main(int argc, char *argv[])
 	/* Master thread waits for other threads to exit */
 	num_thr = odph_thread_join(gbl_args->thread_tbl, num_workers);
 	if (num_thr != num_workers) {
-		ODPH_ERR("Error: worker join failed %i\n", num_thr);
+		ODPH_ERR("Worker join failed %i\n", num_thr);
```

Comment:
":" after "failed" to match the new style.

Reply:
Fixed in v2